core/frontend/src/App.vue

```diff
@@ -331,8 +331,7 @@ export default Vue.extend({
         },
         {
           target: '#tour-center-hook',
-          content: `As stated on our welcome card, one of the first things you should do on your first use of BlueOS
-          is to connect it to the internet.`,
+          content: 'Connect BlueOS to the internet to unlock functionalities.',
```

Comment:
```suggestion
          content: 'Connect BlueOS to the internet to enable online functionalities.',
```
"unlock" comes across (to me) as "if you're offline _we_ block you from some things, but if you turn on the internet we'll allow you to do it", whereas the intent here is more that firmware and version downloads and stuff can't work without some form of online connectivity.

Reply:
Done.